application/src/main/java/de/wuespace/telestion/project/daedalus2/RawFileLogger.java

```diff
@@ -0,0 +1,40 @@
+package de.wuespace.telestion.project.daedalus2;
+
+import de.wuespace.telestion.api.verticle.GenericConfiguration;
+import de.wuespace.telestion.api.verticle.TelestionVerticle;
+import de.wuespace.telestion.api.verticle.trait.WithEventBus;
+import io.vertx.core.Promise;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.file.AsyncFile;
+import io.vertx.core.file.FileSystem;
+import io.vertx.core.file.OpenOptions;
+
+/**
+ * It writes the raw mavlink messages to a file
+ */
+public class RawFileLogger extends TelestionVerticle<GenericConfiguration> implements WithEventBus {
+	AsyncFile file;
+
```

Comment:
```suggestion

	public record Configuration(
		@JsonProperty String inAddress,
		@JsonProperty String filename
	) implements TelestionConfiguration {
	}

```

Lets add some configuration options.